### ISSUE TITLE
compiling la failed

### ISSUE THREAD

**Issue by battilanast** [cpmech/gosl#14]
Hi,
I am so sorry to bother you again.
After formatting my SSD and freshly installed the macOS, I tried again to install gosl on my macbook pro (13" early 2011) which is running macOS Sierra (10.12.6), as described in https://github.com/cpmech/gosl/blob/master/doc/InstallationOnMacOS.md.
I got an Fail during compiling la in the end when I executed `./all.bash` (see attachment). Is the possibly a way to fix this issue?

By the way, there is a typo in 2 HomeBrew and dependencies:
brew install homebrew/science/suitesparse
only worked with:
brew install homebrew/science/suite-sparse

(go version: go version go1.9 darwin/amd64)

![screen_shot1](https://user-images.githubusercontent.com/29484151/29828550-5439667c-8cdd-11e7-8f81-0b96743bea94.png)
![screen_shot2](https://user-images.githubusercontent.com/29484151/29828555-562d2054-8cdd-11e7-8a6f-474cd3f3524e.png)

**Comment by cpmech** [comment]
Thanks again for your interest in Gosl.

I've used very tight tolerances in this test (i.e. 10^{-17}) and it works on some machines (including my MacBook Air, but you know, float-point numbers are weird...). Fixed now: https://github.com/cpmech/gosl/commit/1177b9b65d95ff0a82a87e19ed00bcf742778e75

Thanks also for finding the typo in the installation instructions. Fixed now. By the way, Homebrew suggests using `suite-sparse` now instead of `homebrew/science/suitesparse`
